airmozilla/popcorn/static/popcorn/js/editor.js

```diff
@@ -1,2 +1,29 @@
-// Initialize the editor with the div id and path to Popcorn Editor.
-PopcornEditor.init('editor', '/static/popcorn/PopcornEditor/editor.html');
+$(document).ready(function () {
+  function getSlug() {
+    var config = $('slug');
+    if (config.length) {
+      return config.data('url');
+    } else {
+      console.error('Slug not provided in DOM. Cannot get slug');
+    }
+  }
+
+  PopcornEditor.listen(PopcornEditor.events.loaded, function () {
+    $.get('/popcorn/data/', {
```

Comment:
So that URL there relates to [this](https://github.com/Nolski/airmozilla/blob/editor-load-video/airmozilla/popcorn/urls.py#L13-L15) right?
Once you have a URL written down in a `urls.py` you're supposed to refer to it by the name. It's worth it in the long run. 

This line would be replaced with `$('slug').data('data-url')` or something. 
And in the template you'd do something like this:
```html
<slug data-data-url="{{ url('popcorn:data') }}">
```